test/e2e/dra/utils/deploy.go

```diff
@@ -81,8 +81,13 @@ import (
 )
 
 type Nodes struct {
+	// node names for DRA
```

Comment:
This struct has nothing to do with DRA or device plugins. It could also be used for something else.

Let's call this "the main set of nodes".

Reply:
done